kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -207,7 +207,7 @@ define([
             this.jobManager = jobManager;
             this.toggleTab = toggleTab;
 
-            this.fileTypesDisplay = Util.generateFileTypeMappings(config.typesToFiles);
+            this.fileTypeMapping = config.fileTypeMapping;
```

Comment:
this is generated in the bulk import cell, so we can get it from the config that's passed into the widget.